feature-group-definitions/canvas-context-lost.yml

```diff
@@ -5,7 +5,6 @@ status:
   support:
     chrome: "99"
     edge: "99"
-    firefox: "105"
```

Comment:
Firefox supports only the events on `OffscreenCanvas` and none of the rest of the API.